app/models/session.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Session < ApplicationRecord
+  MAX_AGE = 2.weeks
+
+  include Denormalizable
+  include Environmental
+  include Accountable
+
+  belongs_to :token
```

Comment:
How will this work with SSO, i.e. not tied to a token? Maybe the token association should be optional?

Reply:
But then we'd need to expose sessions so that they can be revoked.

May be easier to simply generate a token as part of the SSO flow.

Reply:
Although, exposing sessions would, in the long run, probably be a good idea from a security perspective. We could show a page on the user's dashboard listing active tokens and sessions, letting them revoke as-needed.

Reply:
Fine with doing that in a follow up PR, though, since it's not directly related to session authn.